cmd/coder/urls.go

```diff
@@ -35,47 +36,66 @@ var urlAccessLevel = map[string]string{
 
 func portIsValid(port string) bool {
 	p, err := strconv.ParseUint(port, 10, 16)
-	if p < 1 {
+	if p < 1 || p > 65535 {
 		// port 0 means 'any free port', which we don't support
 		err = strconv.ErrRange
 	}
 	if err != nil {
-		fmt.Println("Invalid port")
+		flog.Error("Invalid port")
 	}
 	return err == nil
 }
 
 func accessLevelIsValid(level string) bool {
 	_, ok := urlAccessLevel[level]
 	if !ok {
-		fmt.Println("Invalid access level")
+		flog.Error("Invalid access level")
 	}
 	return ok
 }
 
 type createSubCmd struct {
-	access string
+	access  string
+	urlname string
+}
+type editSubCmd struct {
+	createSubCmd
+}
+
+func (sub *editSubCmd) Spec() cli.CommandSpec {
+	return cli.CommandSpec{
+		Name:  "edit",
+		Usage: "<env name> <port> [--access <level>] [--name <name>]",
+		Desc:  "create/update a devurl for external access",
+	}
```

Comment:
This adds a synonym `edit` for `create` purely for usage style, this way users can invoke in ways that show their intention eg:

`coder urls create ...` then
`coder urls edit ...`